<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_novoContato</class>
 <widget class="QDialog" name="fm_novoContato">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog{background:#4169E1;  /* Gradiente de cinza para branco */
    border: none;  /* Remove a borda */
}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>411</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    background: #FFFFFF;
    border: 1px solid #ccc;
    border-radius: 10px;
    padding: 10px;
    box-shadow: 5px 5px 15px rgba(0, 0, 0, 0.4); /* Sombra forte */
}

QGroupBox:title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 5px;
}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_gravar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 1px solid #888888; 
       border-radius: 6px;
       padding: 5px;
       background-color: #4169E1;
      color: #FFFFFF;
		font: bold;</string>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancelar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 1px solid #888888; 
       border-radius: 6px;
       padding: 5px;
       background-color: #FF0000;
      color: #FFFFFF;
	font:bold;</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_email">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>301</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 1px solid #888888; 
       border-radius: 6px;
       padding: 5px;
       background-color: #d3d3d3;
      color: #333333;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_tell">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>301</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 1px solid #888888; 
       border-radius: 6px;
       padding: 5px;
       background-color: #d3d3d3;
      color: #333333;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_nome">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>301</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 1px solid #888888; 
       border-radius: 6px;
       padding: 5px;
       background-color: #d3d3d3;
      color: #333333;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Telefone</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
